templates/template/scripts/regenerate-packages

```diff
@@ -3,6 +3,13 @@ set -e
 
 cd $(dirname $0)
 
+if [[ -z ${BRANCH} ]]; then
+    branch=$(git rev-parse --abbrev-ref HEAD)
+else
+    echo "Using branch ${BRANCH}"
```

Comment:
Would think this echo should be outside of conditional so it always prints ?

Reply:
Sure, will update.